src/components/molecules/EventDisplay/EventDisplay.scss

```diff
@@ -1,134 +0,0 @@
-@import "scss/constants.scss";
```

Comment:
the majority of the file is a copy from `Account.scss`, the rest (`.events-list`) of it I moved to `src/components/molecules/VenueInfoEvents/VenueInfoEvents.scss` where these classes are used.

I coulnd't find any class names used in `EventDisplay.tsx` :grimacing: